README.md

```diff
@@ -527,19 +521,35 @@ def app(environ, start_response):
     return iter([data])
 ```
 
-**Three**: Instrumentation
+**3. Metrics mode and configuration (Gauge)**:
 
-Counters, Summarys and Histograms work as normal.
+When `Gauge` metrics are used, additional tuning needs to be performed.
+First of all, the `gunicorn` configuration file need to include the following function:
```

Comment:
Everyone should do this, not just those using gauges.

Reply:
What benefit does it provide to non `Gauge` metrics, if [its implementation](https://github.com/prometheus/client_python/blob/master/prometheus_client/multiprocess.py#L152) has nothing to do with them?

Reply:
The current implementation doesn't, that doesn't mean it won't in future. In general exposition should be independent of instrumentation anyway, so you should do it in case a dependency uses gauges.

Reply:
I disagree with the first point:

If the implementation changes, affecting other metric types, the `README.md` should reflect that change.

Defining the `mark_process_dead` function just because _"who knows, right? maybe it changes in the future..."_ does not seem like a strong point to me. That same argument could be applied to almost the entire codebase as _"any function may change in the future... right?"_

---
However, I see your second point: a dependency _may_ use a Gauge.